README.md

```diff
@@ -29,7 +29,13 @@ Some browsers (e.g. Firefox) block the font access if the "CORS" headers are not
 To allow access, the domain needs to be whitelisted within the CORS configuration block of the S3 bucket.
 In addition, he ACL settings of the bucket must be set to "public read" so the files can actually be accessed in the first place.
 
-## Local preview
+### cynkra blog, dm and similar repositories
```

Comment:
rephrase the paragraph to mention cynkratemplate?

Reply:
@maelle done!